view/concept-shared.twig

```diff
@@ -42,7 +42,16 @@
       {% spaceless %}
       <div class="row{% if concept.type == 'skosext:DeprecatedConcept' %} deprecated{% endif %} property prop-preflabel">
         <div class="property-label property-label-pref">
-          <h3 class="versal">{% set subPrefLabelTranslation = concept.preferredSubpropertyLabelTranslation(request.lang) %}{% if subPrefLabelTranslation %}{{ subPrefLabelTranslation }}{% else %}{{ 'skos:prefLabel'|trans }}{% endif %}</h3>
+          <h3 class="versal">
+            {% set subPrefLabelTranslation = concept.preferredSubpropertyLabelTranslation(request.lang) %}
+            {% if subPrefLabelTranslation %}
+              {{ subPrefLabelTranslation }}
+            {% elseif custom_labels['skos:prefLabel'] %}
+              {{ custom_labels['skos:prefLabel']['label'][request.contentLang] }}
```

Comment:
Property labels are tooltips are shown in the UI language, not the content language. So the last part of the expression should use UI language.

Reply:
Fixed in a new commit